grails-app/domain/org/pih/warehouse/core/Location.groovy

```diff
@@ -307,6 +307,7 @@ class Location implements Comparable<Location>, java.io.Serializable {
     }
 
     static PROPERTIES = [
+            "id"              : "id",
```

Comment:
I don't love that we're adding ID here. We'd need good justification for including this as I think locationNumber should be used to ensure discoverability and uniqueness.

Reply:
Maybe I'll wait to tech huddle with this. It was added because it was specifically demanded in a ticket.

Reply:
Sounds good. In the meantime, I'll discuss with it Kelsey.